packages/labs/ssr/src/lib/lit-element-renderer.ts

```diff
@@ -81,7 +81,7 @@ export class LitElementRenderer extends ElementRenderer {
     _old: string | null,
     value: string | null
   ) {
-    attributeToProperty(this.element as LitElement, name, value);
+    attributeToProperty(this.element as LitElement, name.toLowerCase(), value);
```

Comment:
Everything in this change looks great! I'm only leaving a comment on this line because I don't have the full picture of what this change does.

My naive thoughts are that the logic for `attributeToProperty` results in calling `_$attributeToProperty` which may call a user defined `converter.fromAttribute`. Does making the name lower case here make sense in case a specific case sensitive name is expected down the line? Or will attributeToProperty always expect a lower case name because attributes from the DOM end up lower cased?

Thank you!

Reply:
I confirmed that the behavior in the client is that it always gets lowercase attributes, regardless of the casing of the attribute name in the original template. In this particular method, we're defining the `attributeChangedCallback` lifecycle and the browser turns all HTML attributes to lowercase so it would be consistent for us to do that in the server as well.

Reply:
Brilliant! Is it worth committing some of that excellent comment as a code comment?

Reply:
Although.. along those lines, perhaps `attributeChangedCallback` should already be getting lowercased names instead. Going to consider moving this to the call site of `attributeChangedCallback` instead.